### DISCUSSION TITLE
Madencilik ve NFT

### DISCUSSION THREAD

**Discussion by muazzezwq** [IBAX-io/go-ibax#467] [Ideas] [answered]
### Clear and concise description of the problem

İllerki süreçlerde NTF Kazanma görevleri gelse iyi olur aslında şahsi fikrim bu yönde...

### Suggested solution

Madencilik yaparsak daha iyi olurdu .

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by Atakanncellik** [comment]
Harika

**Comment by DestinationCrew** [comment]
NFT işi sıkıntı ama madencilik güzel olurdu.

**Comment by Mertdyr35** [comment]
Madencilik üzerine oynanmalı en iyi şey

**Comment by ibaxio** [comment, marked as answer]
In the IBAX Network, the NFT used for mining records the geographical location (latitude and longitude) and the country to which it belongs at the time the NFT is generated.
